tests/test_common.sh.in

```diff
@@ -314,4 +314,14 @@ xsed() {
 	esac
 }
 
+soft_die() {
+	printf '%s' "$1" >&2
+	return 1
+}
+
+die() {
+	soft_die "$1"
+	exit $?
```

Comment:
That's a very complicated and obscure way to return constant `1`. And it's either I don't understand something here or you over-engineered these 2 functions as hell.

Reply:
A part of me probably wanted to customize the return code, so I will make it happen, and it will become normally engineered afterwards.